elements/pfe-styles/src/pfe-typography-classes.scss

```
@@ -1,14 +1,15 @@
 @import "../../pfe-sass/pfe-sass";
 
 // TITLES
-@each $size in (6xl, 5xl, 4xl, 3xl, 2xl, xl, lg, md, sm, xs) {
+@each $size in ("6xl", "5xl", "4xl", "3xl", "2xl", xl, lg, md, sm, xs) {
```

```suggestion
@each $size in (6xl, 5xl, 4xl, 3xl, 2xl, xl, lg, md, sm, xs) {
```

Quotes are no longer needed (yay) because I set up the mixin to quote the input so they are all read as strings now.